Jint/Native/Json/JsonParser.cs

```diff
@@ -879,7 +876,7 @@ private class Extra
         {
             public int? Loc;
             public int[] Range;
-            public string Source;
+            public Script Source;
```

Comment:
If it's a filename then just call it Filename, if it's really source code, is it the full loaded script or just the portion represented by Loc/Range ?